lib/axon/loop/state.ex

```diff
@@ -39,6 +39,9 @@ defmodule Axon.Loop.State do
   # a way in Nx/EXLA to mark `:times` as a static property which is
   # not to be touched at JIT time.
   @enforce_keys [:step_state]
+  @derive {Nx.Container,
+           containers: [:step_state, :epoch, :max_epoch, :iteration, :max_iteration, :metrics],
+           keep: [:times]}
```

Comment:
I don't think you want to keep times. If you keep times, it means every loop interaction has a different `time`, which means it will use a different EXLA cache key and compile it from scratch. You either want to mark it as a container and send it upstream or have the `times` variable be reset and then you roll it back to the current interation once you get the stream back.

Reply:
`EXLA.jit_cached?/3` and `EXLA.stream_cached?/3` are on Nx latest.